packages/genui/playground/src/pages/bench/PhaseTwoReportPage.css

```
@@ -0,0 +1,2191 @@
+  font-weight: 650;
+  line-height: 1.45;
+}
+
+.phaseTwoReportMatchedCoreNotes dd {
+  margin: 5px 0 0;
+  color: var(--phase-report-muted);
+  font-size: 10px;
+  line-height: 1.6;
+}
+
+@keyframes phaseTwoReportMatchedCoreEnter {
```

Stylelint (17.14.1)

[error] Expected keyframe name "phaseTwoReportMatchedCoreEnter" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)